scripts/migrations.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+for MIGRATION in $(ls metadata_store_scripts/V*.sql)
+do
+  git checkout origin/main -- metadata_store_scripts/bootstrap.sql
```

Comment:
You only run this script before opening the PR.

Reply:
I meant the `git checkout` command should be outside the for loop, because I think running this command every time would reset the changes in bootstrap.sql file and we lose whatever update we do in the later part of the `do` block. Please correct me if I am wrong.